ui/components/app/snaps/snap-ui-address/snap-ui-address.tsx

```diff
@@ -20,11 +21,18 @@ export type SnapUIAddressProps = {
   address: string;
   // This is not currently exposed to Snaps.
   avatarSize?: 'xs' | 'sm' | 'md' | 'lg';
+  truncate?: boolean;
+  displayName?: boolean;
+  avatar?: boolean;
 };
 
 export const SnapUIAddress: React.FunctionComponent<SnapUIAddressProps> = ({
   address,
   avatarSize = 'md',
+
+  truncate = true,
+  displayName = false,
```

Comment:
In the previous PR you changed the default to show the name, is it intentional to revert that?

Reply:
No, I think David had that to false when he did this, will default to true

Reply:
Yes, Looking at the definition of the Address component `displayName` defaults to false